Entities/Industry/Tunnel/TunnelTravel.as

```diff
@@ -139,13 +139,18 @@ bool doesFitAtTunnel(CBlob@ this, CBlob@ caller, CBlob@ tunnel)
 
 void Travel(CBlob@ this, CBlob@ caller, CBlob@ tunnel)
 {
-	if (caller !is null && tunnel !is null)
+	CBlob@ thisTunnel = getBlobByNetworkID(this.getNetworkID());
```

Comment:
er, wtf? Surely we should prevent this possibility from happening prior to the `Travel` call. If `this` has a possibility of being a dead blob, then unless we're sure it happens within the same tick, there is a possibility of crashing if the tunnel blob was deleted.

Reply:
This appears to be a problem with how generic buttons work. I haven't checked, but it appears as though it doesn't check if the blob exists when it sends the command when a button is clicked.

Considering no crashes have occurred due to this as far as we know, I think it's fine to just keep this fix.